### PR TITLE
fix: live tail time out is updated

### LINKED ISSUE #1894: Frontend closes live tail connection when there is no activity for 45000 ms
Steps to reproduce
* Go to live tail
* Select an id for the filter.
* When no new data is received for 1 min, the connection is closed.

image

### PR DESCRIPTION
**Pull request by palashgdev**
close #1894

### CODE REVIEW

File: frontend/src/api/logs/livetail.ts [hunk 1]
```diff
@@ -4,14 +4,15 @@ import { ENVIRONMENT } from 'constants/env';
...
 import { EventSourcePolyfill } from 'event-source-polyfill';
 
-export const LiveTail = (queryParams: string): EventSourcePolyfill => {
-	const dict = {
-		headers: {
-			Authorization: `Bearer ${getLocalStorageKey(LOCALSTORAGE.AUTH_TOKEN)}`,
-		},
-	};
-	return new EventSourcePolyfill(
+const TIMEOUT_IN_MS = 120000;
```
Comment (nityanandagohain) [inline frontend/src/api/logs/livetail.ts:7]: lets change it to 10 mins ?

**Comment by nityanandagohain** [comment]
Were you able to test it using the API?

**Comment by palashgdev** [comment]
> Were you able to test it using the API?

how to test via API ?

**Comment by palashgdev** [comment]
though i have confirmed this one with UI

File: frontend/src/api/logs/livetail.ts [hunk 2]
```diff
@@ -4,14 +4,16 @@ import { ENVIRONMENT } from 'constants/env';
...
 
-export const LiveTail = (queryParams: string): EventSourcePolyfill => {
-	const dict = {
-		headers: {
-			Authorization: `Bearer ${getLocalStorageKey(LOCALSTORAGE.AUTH_TOKEN)}`,
-		},
-	};
-	return new EventSourcePolyfill(
+// 10 min in ms
+const TIMEOUT_IN_MS = 10 * 10 * 1000;
```
Comment (ankitnayan) [inline frontend/src/api/logs/livetail.ts:8]: @palashgdev shouldn't this be `10*60*1000`?

**Comment by ankitnayan** [comment]
@palashgdev @nityanandagohain this does not seem to fix the issue. Please have a look again.

**Comment by palashgdev** [comment]
@nityanandagohain config is right but what could be the possible reason https://github.com/Yaffle/EventSource/blob/master/src/eventsource.js#LL742C50-L742C66 ?

**Review by nityanandagohain — APPROVED** [review]

**Pull request merged**